### DISCUSSION TITLE
Reanimated spring preview/calculators/visualizer

### DISCUSSION THREAD

**Discussion by Noitidart** [software-mansion/react-native-reanimated#2833] [General]
Anyone know of a good spring/preview calculator which has the props that much reanimated? (damping, mass, stiffness, overshootClamping, restDisplacementThreshold, restSpeedThreshold)

**Comment by headfire94** [comment]
this one can be helpful https://github.com/mohit23x/reanimated-config-visualizer

**Reply by Noitidart** [reply]
Thank you!

**Comment by Vimal1464** [comment]
can anyone help me with this
https://github.com/software-mansion/react-native-reanimated/issues/6074

**Comment by MatiPl01** [comment]
It seems that the solution for the problem mentioned in this discussion is already provided in the [comment](https://github.com/software-mansion/react-native-reanimated/discussions/2833#discussioncomment-8631372) above. I am closing the issue.

**Reply by Vimal1464** [reply]
ye please close
